x/pairing/keeper/grpc_query_effective_policy.go

```diff
@@ -22,7 +22,7 @@ func (k Keeper) EffectivePolicy(goCtx context.Context, req *types.QueryEffective
 	if err != nil {
 		return nil, err
 	}
-	strictestPolicy, _, err := k.GetProjectStrictestPolicy(ctx, project, req.SpecID)
+	strictestPolicy, _, err := k.GetProjectStrictestPolicy(ctx, project, req.SpecID, uint64(ctx.BlockHeight()))
```

Comment:
if you use here current height, it means that if I run the effective policy query with a different height (using the --height flag), it won't work right? (not sure, but worth a check)

Reply:
when using --height the ctx.blockheight changes accordingly